apps/swirl-docs/src/documents/components/action-list.mdx

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
Should these files be placed alongside the components in `packages/swirl-components`?

Reply:
when we put the mdx files into the `swirl-components` package we need to change the build and deployment process a little bit. Do you think it would be much better to have them along the components?

Reply:
It would be nice to have everything in one place. But can wait.